extensions/open-folders/src/folderSpeedDial.tsx

```diff
@@ -0,0 +1,61 @@
+import { open, getPreferenceValues, Grid, ActionPanel, Action } from "@raycast/api";
+
+interface Preferences {
+  dirOne: string;
+  dirTwo: string;
+  dirThree: string;
+  dirFour: string;
+  dirFive: string;
+  dirSix: string;
+  dirSeven: string;
+  dirEight: string;
+}
+
+const preferences = [
+  getPreferenceValues<Preferences>().dirOne,
+  getPreferenceValues<Preferences>().dirTwo,
+  getPreferenceValues<Preferences>().dirThree,
+  getPreferenceValues<Preferences>().dirFour,
+  getPreferenceValues<Preferences>().dirFive,
+  getPreferenceValues<Preferences>().dirSix,
+  getPreferenceValues<Preferences>().dirSeven,
+  getPreferenceValues<Preferences>().dirEight,
+];
```

Comment:
```suggestion
const preferenceValues = getPreferenceValues<Preferences.FileSpeedDial>;

const preferences = [
  preferenceValues.dirOne,
  preferenceValues.dirTwo,
  preferenceValues.dirThree,
  preferenceValues.dirFour,
  preferenceValues.dirFive,
  preferenceValues.dirSix,
  preferenceValues.dirSeven,
  preferenceValues.dirEight,
];
```
Maybe rely on types from `raycast-env.d.ts` instead (and do that in other files since it's autogenerated)

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.